.github/workflows/go-presubmit.yml

```diff
@@ -23,25 +19,19 @@ jobs:
     steps:
       - name: checkout code
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 1
```

Comment:
The default `fetch-depth` is 0. https://github.com/actions/checkout/blob/ac593985615ec2ede58e132d2e21d2b1cbd6127c/README.md?plain=1#L78